application/src/main/java/org/openhds/mobile/activity/NavigateActivity.java

```diff
@@ -322,12 +340,48 @@ private void hierarchySetup() {
             showDetailFragment();
             detailToggleFragment.setButtonHighlighted(true);
         }
+        //populate recent forms
+        populateFormsForPath();
```

Comment:
Unnecessary comment? Usually when the comment is so close to the method name it's an indication that you might want to create a more appropriate name for the method.